File: js/01-say-hello/01.mjs
import * as readline from 'node:readline/promises';
import { stdin as input, stdout as output } from 'node:process';

const rl = readline.createInterface({
  input,
  output,
});

const name = await rl.question("What is your name? ")

console.log(`Hello, ${name}, nice to meet you!`)
rl.close();

File: c/01-say-hello/01.c
# include <stdio.h>
#include <string.h>

int main() {
  char name[50];
  printf("What is your name? ");
  fgets(name, 50, stdin);
  name[strcspn(name, "\n")] = 0;
  printf("Hello, %s, nice to meet you", name);
  return 0;
}
